File: docs/deployment/codesnippet/CSharp/accessing-local-and-remote-data-in-clickonce-applications_1.cs
            if (ApplicationDeployment.IsNetworkDeployed)
            {
                try
                {
                    using (StreamReader sr = new StreamReader(ApplicationDeployment.CurrentDeployment.DataDirectory + @"\CSV.txt"))
                    {
                        MessageBox.Show(sr.ReadToEnd());
                    }
                }
                catch (Exception ex)
                {
                    MessageBox.Show("Could not read file. Error message: " + ex.Message);
                }
            }

File: docs/deployment/codesnippet/VisualBasic/accessing-local-and-remote-data-in-clickonce-applications_1.vb
        If (ApplicationDeployment.IsNetworkDeployed) Then
            Dim SR As StreamReader = Nothing

            Try
                SR = New StreamReader(ApplicationDeployment.CurrentDeployment.DataDirectory & "\CSV.txt")
                MessageBox.Show(SR.ReadToEnd())
            Catch Ex As Exception
                MessageBox.Show("Could not read file.")
            Finally
                SR.Close()
            End Try
        End If